ui/app/components/ui/button/buttons.scss

```diff
@@ -243,11 +241,12 @@ input[type="submit"][disabled] {
 }
 
 button.primary {
+  @include H4;
```

Comment:
This is different than the original but i can't find a place where we use a button with just a 'primary' class.